api/user/author/authors.py

```diff
@@ -45,6 +45,13 @@ def get_single_author(author_id: str):
     return found_author.getJSON()
 
 
+@authors_bp.route("/authenticated_user_id", methods=["GET"])
+@login_required
+def manpreet():
+    if current_user.is_authenticated:
+        return {"id": current_user.id}
+
+
```

Comment:
is this still in use? haven't found any reference

Reply:
Gonna be using it in the future to get the user id of the user that has logged in

Reply:
I guess we do need that to get their feed...:facepalm: